java/tools/src/test/java/org/apache/tsfile/tools/TsfiletoolsTest.java

```diff
@@ -95,6 +103,35 @@ public void genSchemaFile() {
     }
   }
 
+  public void genWrongCsvFile(int rows) {
+
+    try (BufferedWriter writer = new BufferedWriter(new FileWriter(wrongCsvFile))) {
+      writer.write("time,tmp1,tmp2,tmp3,tmp4,tmp5");
+      writer.newLine();
+      Random random = new Random();
+      long timestamp = System.currentTimeMillis();
+
+      for (int i = 0; i < rows; i++) {
+        timestamp = timestamp + i;
+        String tmp1 = "s1";
+        float tmp2 = random.nextFloat();
+        float tmp3 = random.nextFloat();
+        float tmp4 = random.nextFloat();
+        float tmp5 = random.nextFloat();
+        if (i % 99 == 0) {
+          writer.write(
+              timestamp + "aa" + "," + tmp1 + "," + tmp2 + "," + tmp3 + "," + tmp4 + "," + tmp5);
+        } else {
+          writer.write(timestamp + "," + tmp1 + "," + tmp2 + "," + tmp3 + "," + tmp4 + "," + tmp5);
+        }
+
+        writer.newLine();
+      }
+    } catch (IOException e) {
+      e.printStackTrace();
```

Comment:
May use `fail(e.getMessage())` or simply throw the exception, since this is a test.